orderfilter/validate_js_test.go

```diff
@@ -0,0 +1,38 @@
+// +build js, wasm
+
+package orderfilter
+
+var (
+	testFilterValidateOrderExpectedErrors = [][]string{
```

Comment:
IMO This approach makes the tests too brittle. They will break if you add a new test case, change the order of test cases, or comment out some test cases. Instead of creating an implicit dependence on the ordering of test cases and the corresponding errors could we make each expected error a named variable? Something like:

```go
// +build !js

var (
    missingMakerAddressErrors = []string{
        "put the expected errors for Go here",
    }
)
```

```go
// +build js,wasm

var (
    missingMakerAddressErrors = []string{
        "put the expected errors for JavaScript/Wasm here",
    }
)
```

```diff
// In each test case, reference the expected errors by name...
{
	note:              "order with missing makerAddress",
	chainID:           constants.TestChainID,
	customOrderSchema: DefaultCustomOrderSchema,
	orderJSON:         []byte{ /* ... */ },
+	expectedErrors:    missingMakerAddressErrors,	
},
```

This way the tests will not break if you change the order of test cases, comment some out, or add new ones. And if you add the expected errors for one environment but not the other, the compiler will yell at you.